Checkout/Checkout/Source/Model/Card+Scheme.swift

```diff
@@ -120,8 +122,11 @@ extension Card {
       switch self {
```

Comment:
Small suggestion if we can inject `cardNumberLength` then it would be small change then no need for extra code like `public var rawValue: String`, `allCases` and so on
```suggestion
func cvvLengths(with cardNumberLength: Int = 0) { 
      switch self {
```

Reply:
I have had this discussion before going down this path in our standup (I think you were present), and presented options, with @harry-brown-cko suggesting associated enum may be the way to go.

I have also researched your suggestion prior to that and it gets messy quickly because the CardValidator is inside the Checkout framework and as such any changes to its interface will have additional impact and increase complexity in multiple places.

Having tried both ways, I do agree with @harry-brown-cko 's opinion when we decided to go with this, that this is actually the easier to digest solution and is the cleanest solution to implement. Extending an interface for validator would have way more consequences than are apparent on first sight.